src/azure-cli/azure/cli/command_modules/acr/_client_factory.py

```diff
@@ -43,7 +43,7 @@ def cf_acr_webhooks(cli_ctx, *_):
 
 
 def cf_acr_tasks(cli_ctx, *_):
```

Comment:
@shahzzzam the clients `cf_acr_registries_tasks` and `cf_acr_runs` also belong to our API (not ACR's). Please add the hardcoded API version to their methods as well.